metadata-proxy/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/google-containers/debian-base-amd64:0.1
+LABEL maintainer "qlee@google.com"
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y \
```

Comment:
Interesting.

no cleanup:        fb786fa5fc50        112 MB
"clean-install":        730191ef3d6f        61.94 MB
"apt-get clean -y":        7f7e7eac1fb7        112 MB
gcr.io/google-containers/debian-base-amd64:        61a750f1056b        40.64 MB

The "rm -rf" seems to be removing about 50MB of junk. I'm convinced.